packages/hash/api/src/graph/workspace-types.ts

```diff
@@ -22,12 +26,22 @@ export let WORKSPACE_TYPES: {
     orgName: PropertyTypeModel;
     orgSize: PropertyTypeModel;
     orgProvidedInfo: PropertyTypeModel;
+
+    // OrgMembership-related
```

Comment:
Can we avoid using abbreviations for names? (use Organization instead of Org)

Reply:
I did this for consistency with the existing `OrgModel` class. Are you therefore suggesting I change the name of both? In which case this would substantially affect the diff of this PR, so may be best addressed in follow-up

Reply:
Sorry, I wasn't clear here. I don't suggest changing it now but in a follow-up. This is definitely out of scope of this PR 🙂

Reply:
Cool, have added [the follow-up task](https://app.asana.com/0/1201095311341924/1202980861294692/f)

Reply:
Great, thanks!

Reply:
I [don't think we should do this](https://app.asana.com/0/0/1202980861294692/1202999360660298/f) (internal link).